examples/ota-requestor-app/esp32/README.md

```diff
@@ -40,20 +40,13 @@ After commissioning is successful, announce OTA provider's presence using
 chip-tool. On receiving this command OTA requestor will query for OTA image.
 
 ```
-./out/debug/chip-tool otasoftwareupdaterequestor announce-ota-provider 12345 0 0 12346 0
+./out/debug/chip-tool otasoftwareupdaterequestor announce-ota-provider 12345 0 0 0 12346 0
```

Comment:
What parameter has been added in this command?

Reply:
endpoint